### ISSUE TITLE
Release v1.34.3

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#2042]
:robot: A dedicated issue to run tests and deploy release [v1.34.3](https://github.com/deckhouse/deckhouse/milestone/80).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.34.3 | test-v1.34.3*` or a branch `main | release-1.34`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22 | 1.23`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.34.3 | test-v1.34.3*` or a branch `main | release-1.34`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21 using image built from main branch.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.34
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [9e8e0f](https://github.com/deckhouse/deckhouse/commit/9e8e0f1082ada80987c0c4f7e8ed8f23b21cdd53) in branch **main**:
```
[prometheus] Update to 2.37.0 (LTS) (#2034)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [a640ac](https://github.com/deckhouse/deckhouse/commit/a640acd01e724ee731a306c8481807ba137fa258) in branch **main**:
```
[candi] fix docker config creation (#2044)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.34.3** is created.
